process/subprocesses/ghsci.py

```diff
@@ -757,7 +751,9 @@ def _region_data_setup(
                     return None
                 data_dictionary = datasets[data][region_config[data]].copy()
             else:
-                if data == 'urban_region' and region_config[data] is None:
+                if data == 'urban_region' and (

```

Comment:
Was there a logic problem with the way this was before? If this fixes a known problem, then I think that is good, but if it is just rephrasing for sake of it, I would not make the change.